plone/namedfile/scaling.py

```diff
@@ -66,15 +67,15 @@ def absolute_url(self):
         return self.url
 
     def srcset_attribute(self):
-        srcset_attr = []
+        _srcset_attr = []
         extension = self.data.contentType.split('/')[-1].lower()
         for scale in self.srcset:
-            srcset_attr.append(u'{0}/@@images/{1}.{2} {3}x'.format(
+            _srcset_attr.append(u'{0}/@@images/{1}.{2} {3}x'.format(
                 self.context.absolute_url(),
                 scale['uid'],
                 extension,
                 scale['scale']))
-        srcset_attr = ', '.join(srcset_attr)
+        srcset_attr = ', '.join(_srcset_attr)
```

Comment:
True. Again, my linter complained, and I think this formatting rule is ok. The linter said, here the type of the variable is changed from a list to a string.
I don't have a strong opinion on this. Should I change it back?